sources/Core/Core/Abstractions/IGLContext.cs

```diff
@@ -5,34 +5,29 @@ namespace Silk.NET.Core;
 /// <summary>
 /// Represents an OpenGL context.
 /// </summary>
-public interface IGLContext : INativeContext, IDisposable
+public interface IGLContext : INativeContext
 {
     /// <summary>
     /// Whether the context is current on this thread.
     /// </summary>
-    bool IsCurrent { get; }
+    bool IsCurrent { get; set; }
```

Comment:
so can we make this context current on this thread?

Reply:
actually better question given that there was a makecurrent previously. What happens if I try and set this to false?

Reply:
IsCurrent = true is MakeCurrent. The context will be made current on the calling thread.
IsCurrent = false is ClearCurrent (or Clear, don't remember what this was called). The context will be cleared from the calling thread.